app/jobs/patient_import_job.rb

```diff
@@ -0,0 +1,5 @@
+class PatientImportJob < ApplicationJob
+  def perform(params:, facility:, admin:)
+    PatientImport::Importer.new(params: params, facility: facility, admin: admin).import
```

Comment:
Do you think its worth adding a log here with `facility_id`, `patient_id`, and a param to indicate success/failure?
This will help us look up aggregate data for an input file via datadog.

Reply:
We're sticking to verification of aggregates by inspecting sidekiq queues and using the console. I think this should suffice for now. 

Actually, we're raising exceptions on failure, so we could confidently log successes along with audit logs. Yup, may be worth it to avoid having to trawl through audit logs to inspect import activity. Will do.